README.md

```diff
@@ -16,34 +16,45 @@ Documentation can be found [here](https://eosio.github.io/eosjs)
 
```

Comment:
Under the installation header a few lines above, there are only NodeJS installation instructions, you might want to either remove that header (### NodeJS) or add that it also works for the browser with projects using npm